architecture/wm_ciscat/Readme.md

```diff
@@ -0,0 +1,25 @@
+<!---
+Copyright (C) 2015-2021, Wazuh Inc.
+Created by Wazuh, Inc. <info@wazuh.com>.
+This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+-->
+
+# Wazuh module: CIS-CAT architecture
+## Index
+1. [Purpose](#purpose)
+2. [Sequence Diagram](#sequence-diagram)
+
+## Purpose
+The **CIS-CAT** Wazuh module integrates CIS benchmark assessments into Wazuh agents and reports the results of each scan in the form of an alert. The module requires the use of **CIS-CAT Pro**, an external tool developed for scanning target systems and generating a report comparing the system settings to the CIS benchmarks.
+
+## Sequence Diagram
+The provided sequence diagram shows the basic flow of Wazuh's **CIS-CAT** module. The main steps are:
+
+1. The **SCIS-CAT** module is executed according to the configuration provided in `ossec.conf`. The module then sleeps, based on the specified timestamp.
```

Comment:
- `SCIS-CAT` -> `CIS-CAT`

Reply:
- `The module then sleeps, based on the specified timestamp.` -> Remove this sentence, it is described in the step 2.